pkg/tfpfbridge/naming.go

```diff
@@ -22,8 +22,31 @@ import (
 	"github.com/pulumi/pulumi/sdk/v3/go/common/tokens"
 
 	"github.com/pulumi/pulumi-terraform-bridge/pkg/tfpfbridge/internal/convert"
+	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfgen"
 )
 
+type PrecisePropertyNames struct {
+	renames tfgen.Renames
+}
+
+var _ convert.PropertyNames = (*PrecisePropertyNames)(nil)
+
+func (s *PrecisePropertyNames) PropertyKey(
+	typeToken tokens.Token,
+	property convert.TerraformPropertyName,
+	_ tftypes.Type,
+) resource.PropertyKey {
+	if renamedProps, ok := s.renames.RenamedProperties[typeToken]; ok {
+		// Perhaps this table should be inverted upfront.
```

Comment:
should this be a TODO or an issue?

Reply:
Let me fix it up right away, it's small.